README.md

```diff
@@ -61,14 +61,19 @@ python -m venv .venv
 ### Dependencies
 
 To use the linker in OpenTimelineIO `openassetio` needs to be installed.
-At present, this requires installation from source:
+At present, this requires installation from source of the
+`feature/181-coreCppBuild` branch.
```

Comment:
[fix] OpenAssetIO-MediaCreation is also a dependency now

Reply:
Added [here](https://github.com/TheFoundryVisionmongers/otio-openassetio/pull/5/commits/26c66af609ca22ebab79f81958ac906a7b71e3aa).